src/Web/WebSPA/Client/modules/catalog/catalog.component.ts

```diff
@@ -56,6 +56,9 @@ export class CatalogComponent implements OnInit {
 
     onFilterApplied(event: any) {
         event.preventDefault();
+        
+        this.brandSelected = this.brandSelected && this.brandSelected.toString() != "null" ? this.brandSelected : null;
```

Comment:
As a JS newbie I would've thought this is redundant, so it looks like I have to know better 😅